src/audiovideofacade/DefaultAudioVideoFacade.ts

```diff
@@ -311,20 +311,20 @@ export default class DefaultAudioVideoFacade implements AudioVideoFacade {
     this.trace('unsubscribeFromActiveSpeakerDetector');
   }
 
-  async listAudioInputDevices(): Promise<MediaDeviceInfo[]> {
-    const result = await this.deviceController.listAudioInputDevices();
+  async listAudioInputDevices(forceUpdate: boolean = false): Promise<MediaDeviceInfo[]> {
+    const result = await this.deviceController.listAudioInputDevices(forceUpdate);
     this.trace('listAudioInputDevices', null, result);
     return result;
   }
 
-  async listVideoInputDevices(): Promise<MediaDeviceInfo[]> {
-    const result = await this.deviceController.listVideoInputDevices();
+  async listVideoInputDevices(forceUpdate: boolean = false): Promise<MediaDeviceInfo[]> {
+    const result = await this.deviceController.listVideoInputDevices(forceUpdate);
     this.trace('listVideoInputDevices', null, result);
```

Comment:
These `trace` calls should have `forceUpdate` as the second argument, throughout.

Reply:
Added.